lib/hw_unique_key/Kconfig

```
@@ -4,6 +4,15 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+config HW_UNIQUE_KEY_LOAD
+	bool "Load Hardware Unique key on boot"
+	depends on HAS_HW_NRF_ACL
+	depends on HAS_HW_NRF_CC310
+	depends on NRF_CC3XX_PLATFORM
+	select FPROTECT if HAS_HW_NRF_ACL
+	imply FPROTECT_ENABLE_NO_ACCESS if HAS_HW_NRF_ACL
+	imply NRFX_NVMC if HAS_HW_NRF_ACL
```

This setting has a `depends on HAS_HW_NRF_ACL` 3 lines above, so there should be no need for the `if HAS_HW_NRF_ACL`.

```suggestion
	select FPROTECT
	imply FPROTECT_ENABLE_NO_ACCESS
	imply NRFX_NVMC
```

---

@tejlmand thanks, done